container/check_container.go

```diff
@@ -55,7 +88,7 @@ func (c *containerCheck) Run(ctx context.Context) (certification.Results, error)
 
 		override, err := lib.GetContainerPolicyExceptions(ctx, p)
 		if err != nil {
-			return certification.Results{}, fmt.Errorf("%w: %s", preflighterr.ErrCannotResolvePolicyException, err)
+			return policy.PolicyNone, []check.Check{}, fmt.Errorf("%w: %s", preflighterr.ErrCannotResolvePolicyException, err)
```

Comment:
Since Policy is just a type alias, and not a typedef, you can just return "" here without the addition of
"PolicyNone".

I would prefer that.

Reply:
fixed